tailwind.config.js

```diff
@@ -2,7 +2,51 @@
 module.exports = {
   content: ['./src/**/*.{html,ts}', 'src/**/*.{css,scss}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        accent: '#3545E9',
+        warning: '#FEFD54',
+        success: '#63C995',
+        error: '#E23D69',
+        primary: '#222328',
+        secondary: '#969AB0',
+        bright: '#FFFFFF',
+        muted: '#7C9CBF',
+        shade: '#DBE2EA',
+      },
+      fontFamily: {
+        'ibm-plex-sans-regular': ["IBMPlexSans-Regular"],
+        'ibm-plex-sans-semi-bold': ["IBMPlexSans-SemiBold"]
+      },
+      fontSize: {
+        x1: '72px',
+        x2: '64px',
+        x3: '56px',
+        x4: '34px',
+        x5: '28px',
+        x6: '24px',
+        paragraphX1: '16px',
+        paragraphX2: '14px',
+        label: '12px'
```

Comment:
same here: we don't need to set a new config if Tailwind already provides us with the needed values. For example for font sizes: label = text-xs, paragraphX2 = text-sm, etc. 

You can think this way: let's say you need font-size 72px, you go to tailwind docs https://tailwindcss.com/docs/font-size, check if there are required values -> yes, it's text-7xl, so you don't need to add it to config.

I'd rather not add all properties at once, but on demand while working on the project. In that case, you'll see if you need to add some config, or you can use a custom approach, like text-[72px].

Reply:
You also can consider adding a component wrapper that takes as input type of typography (h1-h6, label, etc.), handles font-size, line-height etc according to the design for the specific type. In that case you won't need to provide these styles in each p, h1-h6 etc.